workflow/tests/base_results/results_simulations_energy.csv

```diff
@@ -278,12 +278,17 @@ base-hvac-furnace-oil-only.xml,54.479,54.479,30.812,30.812,0.0,23.667,0.0,0.0,0.
-base-hvac-ground-to-air-heat-pump-backup-stove.xml,40.439,40.439,40.438,40.438,0.0,0.0,0.0,0.0,0.0,0.0,5.675,1.353,0.0,0.0,2.335,0.926,9.014,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,2.07,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
-base-hvac-ground-to-air-heat-pump-cooling-only.xml,33.34,33.34,33.34,33.34,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,2.376,0.934,9.061,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,1.902,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
-base-hvac-ground-to-air-heat-pump-detailed-geothermal-loop.xml,40.721,40.721,40.721,40.721,0.0,0.0,0.0,0.0,0.0,0.0,6.026,1.41,0.0,0.0,2.186,0.95,9.014,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,2.069,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
-base-hvac-ground-to-air-heat-pump-heating-only.xml,36.784,36.784,36.784,36.784,0.0,0.0,0.0,0.0,0.0,0.0,5.314,1.265,0.0,0.0,0.0,0.0,8.992,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,2.146,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
-base-hvac-ground-to-air-heat-pump.xml,40.055,40.055,40.055,40.055,0.0,0.0,0.0,0.0,0.0,0.0,5.348,1.273,0.0,0.0,2.352,0.932,9.013,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,2.072,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
```

Comment:
I compared the IDFs between this branch and master and found that the single speed ghp test results changes are caused by:
1. The rated air flow rate change, this is a bugfix that we used the design cfm as its rated before, and now it's fixed by using the rated cfm/ton * capacity(ton)
2. The hvac sizing changes that caused a small diff in cooling cfm, I think it's a combined impact of:  
     (1) introduced the heating performance curve adjustment to convert design to rated conditions, which was missing before 
    (2) In calculating the cooling performance curve, we precalculated the cooling air flow using sensible cooling loads and cfm/ton, it's now changed to use total cooling loads.

Reply:
Yes I think this is something we should discuss, I'm pretty sure the ~20% increase is caused by the PLR curve we introduced, I was trying to justify the diff between "simple" and "advanced", so I made them as consistent as possible except for they're using two different E+ coils.

![image](https://github.com/user-attachments/assets/6061d7a2-7aad-4c9a-bf82-43352a200d29)

This is the results I got from simple model + reverting the PLR curve change, current simple model, and current advanced model.